tests/rbd_mirror/test_mirror_move_primary_trash_restore.py

```diff
@@ -0,0 +1,102 @@
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verification of primary image having snap and clone move to trash and
```

Comment:
Maybe change this to "Verification of trash restore on an image with snap and clone in primary cluster to check that mirror relationship is intact post restore" so that the functionality being tested is clearer.

Reply:
done